autofs-5.0.7 - fix nobind man page description

From: Ian Kent <raven@themaw.net>

Update auto.master(5) to better describe the behavior of the "nobind"
option when used with direct mounts maps.
---

 CHANGELOG            |    1 +
 man/auto.master.5.in |   10 +++++++---
 2 files changed, 8 insertions(+), 3 deletions(-)


--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -116,6 +116,7 @@
 - fix null map entry order handling.
 - allow non root user to check status.
 - fix map entry duplicate offset detection.
+- fix nobind man page description.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/man/auto.master.5.in
+++ autofs-5.0.5/man/auto.master.5.in
@@ -143,9 +143,13 @@ can't be mounted.
 This is an autofs specific option that is a pseudo mount option and
 so is given without a leading dash. It may be used either in the master
 map entry (so it effects all the map entries) or with individual map
-entries to prevent bind mounting of local NFS filesystems. Bind mounting
-of NFS file systems can also be prevented for specific map entrys by
-adding the "port=" mount option to the entries.
+entries to prevent bind mounting of local NFS filesystems. For direct
+mount maps the option is only effective if specified on the first direct
+map entry and is applied to all direct mount maps in the master map. It
+is ignored if given on subsequent direct map entries. It may be used
+on individual map entries of both types. Bind mounting of NFS file
+systems can also be prevented for specific map entrys by adding the
+"port=" mount option to the entries.
 .TP
 .I "\-r, \-\-random-multimount-selection"
 Enables the use of ramdom selection when choosing a host from a
